Tests/MSFT_xDhcpServerAuthorization.Tests.ps1

```diff
@@ -0,0 +1,134 @@
+[CmdletBinding()]
+param()
+
+if (!$PSScriptRoot) # $PSScriptRoot is not defined in 2.0
+{
+    $PSScriptRoot = [System.IO.Path]::GetDirectoryName($MyInvocation.MyCommand.Path)
+}
+
+$ErrorActionPreference = 'Stop'
```

Comment:
@TravisEz13 So remove the `if (!$PSScriptRoot)` scriptblock, but leave the `$ErrorActionPreference = 'Stop'` in there?